src/stores/infra/postgres/client/dialer.go

```diff
@@ -0,0 +1,140 @@
+package client
+
+import (
+	"context"
+	gotls "crypto/tls"
+	"encoding/binary"
+	"fmt"
+	"io"
+	"net"
+
+	"github.com/consensys/quorum-key-manager/pkg/errors"
+	"github.com/consensys/quorum-key-manager/pkg/tcp"
+	"github.com/consensys/quorum-key-manager/pkg/tls"
+)
+
+type SSLDialer struct {
```

Comment:
With even SSL 3.0 deprecated, is it good to have an implem for this?

Reply:
it seems the TLSDIaler uses the SSLDIaler:
```
&TLSDialer{
		Dialer: &tls.Dialer{
			Dialer: &SSLDialer{
				Dialer: &net.Dialer{
					Timeout:   cfg.DialTimeout,
					KeepAlive: cfg.KeepAliveInterval,
				},
			},
			TLSConfig: tlsConfig,
		},
		verifyCAOnly: verifyCAOnly,
	}
```

Reply:
Oh, ok, I got it I think.
It actually just needs a wrapper around a `tcp.Dialer`, so maybe rename the SSLDialer to `pgTLSDialer`?
Also, we don't need it outside this package, right? So I guess private visibility would be better?

Reply:
comments applied